bundles/EcommerceFrameworkBundle/Command/IndexService/BootstrapCommand.php

```diff
@@ -35,7 +35,8 @@ protected function configure()
             ->addOption('create-or-update-index-structure', null, InputOption::VALUE_NONE, 'Use to create or update the index structure')
             ->addOption('update-index', null, InputOption::VALUE_NONE, 'Use to rebuild the index data')
             ->addOption('object-list-class', null, InputOption::VALUE_REQUIRED, 'The object list class to use', '\\Pimcore\\Model\\DataObject\\Product\\Listing')
-            ->addOption('list-condition', null, InputOption::VALUE_OPTIONAL, 'An optional condition for object list', '');
```

Comment:
sorry, one more thing: why this option was removed? 
I think we should readd it.

Reply:
Yes! Done.